<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QWidget" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>229</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>math-teacher</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>225</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timeEdit">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fromEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="operationLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="goButton">
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fromEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>goButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>goButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>goButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
